### DISCUSSION TITLE
Is there any plan to update the Nuget version recently?

### DISCUSSION THREAD

**Discussion by snowiceyeah** [MahApps/MahApps.Metro.IconPacks#318] [General]

**Comment by punker76** [comment]
I currently work on an update and plan to publish this in the next days.

**Comment by hupo376787** [comment]
Any update, thx?

**Reply by punker76** [reply]
I'm still trying to enable .Net 6, but it's not working so well at the moment. The msbuild sdk is still a problem. Maybe I will keep then it by .Net 5

**Comment by snowiceyeah** [comment]
Any update?

**Reply by timunie** [reply]
Open Soruce -> It's done when it's done. Feel free to consume the source directly if needed.

**Comment by punker76** [comment]
Update: I'm still trying to move to a more current .Net framework while still enabling UWP with the msbuild sdk. But it looks good and hopefully won't take that long. Thanks for your patience! ❤️❤️❤️

**Reply by snowiceyeah** [reply]
thx

**Comment by punker76** [comment]
👋 v5 is out
